### DISCUSSION TITLE
Is `Diagnostics.InlayHints.Designators` key is valid?

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [clangd/clangd#1478] [Q&A] [answered]
Is `Diagnostics.InlayHints.Designators` key is valid? I can't find it documented [here](https://clangd.llvm.org/config.html). I am asking because of [this](https://github.com/SchemaStore/schemastore/pull/2760#event-8366464646) PR.

**Comment by HighCommander4** [comment, marked as answer]
It's valid, just missing in the documentation (https://github.com/clangd/clangd/issues/1472)
